app/src/main/java/com/jerboa/MainActivity.kt

```diff
@@ -493,7 +493,11 @@ class MainActivity : AppCompatActivity() {
                         ),
                     ) {
                         val args = Route.PostArgs(it)
-                        SwipeToNavigateBack(appState::navigateUp) {
+
+                        SwipeToNavigateBack(
+                            appSettings.useSwipeBack,
```

Comment:
Make it a enum, later when full swipe gestures gets added we can just add it to this enum.